.github/config.yml

```diff
@@ -0,0 +1,16 @@
+# Comment to be posted on PRs from first time contributors in your repository
+newPRWelcomeComment: >
```

Comment:
so this config.yml is great. Can we move it to https://github.com/flyteorg/boilerplate. This allows us to clone the newPRWelcomeComment and firstPRMergeComment to all other flyte repos